editoast/editoast_models/src/lib.rs

```diff
@@ -1,16 +1,23 @@
-use diesel_async::{pooled_connection::deadpool::Pool, AsyncPgConnection};
+use db_connection_pool::DatabasePoolError;
+use diesel_async::pooled_connection::deadpool::Pool;
+use diesel_async::AsyncPgConnection;
 
 pub mod db_connection_pool;
 pub mod tables;
 
+pub use db_connection_pool::DbConnection;
 pub use db_connection_pool::DbConnectionPoolV2;
 
-pub type DbConnection = AsyncPgConnection;
-pub type DbConnectionPool = Pool<DbConnection>;
+pub type DieselConnection = AsyncPgConnection;
```

Comment:
Why is this type still `pub`? We'd like to only expose `DbConnection` outside of the crate. Am I missing something?